build-system/compile/helpers.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const minimist = require('minimist');
+const sourcemaps = require('gulp-sourcemaps');
+const {VERSION: internalRuntimeVersion} = require('./internal-version');
+
+const argv = minimist(process.argv.slice(2));
+
+function getSourceMapBase() {
+  if (!argv.fortesting) {
```

Comment:
This implies `argv.sourcemap_url` can only be used with `argv.fortesting`. Could we move `if (argv.sourcemap_url)` before this condition?

Reply:
Agree, there's no reason `sourcemap_url` shouldn't work with `--fortesting`.

Reply:
Done.